### ISSUE TITLE
InvalidOperation on v11.2.3291

### ISSUE THREAD

**Issue by officeSpacePirate** [OctopusDeploy/OctopusClients#584]
With no changes to a powershell script I have in a step template using Octopus.Client, I have started getting the following error coming from instantiating `Octopus.Client.OctopusRepository($endpoint)`:
```
InvalidOperation: Exception calling ".ctor" with "1" argument(s): "The type initializer for 'Octopus.Client.Serialization.JsonSerialization' threw an exception."
```

Reverting to v11.2.3274 works as expected. Is there a breaking change that should be expected going forward or is this a bug?

Thanks

**Comment by thijsk** [comment]
Also ran into this problem, was 11.3.3291 withdrawn from nuget.org ?

**Comment by andrew-at-octopus** [comment]
Yes, there was a bug introduced in version 11.2.3283, which was fixed in 11.2.3319.
The offending versions were not released (or removed from NuGet).
A new version should be available shortly
